units/src/amount/signed.rs

```diff
@@ -158,7 +158,7 @@ impl SignedAmount {
     pub fn from_str_in(s: &str, denom: Denomination) -> Result<SignedAmount, ParseAmountError> {
         match parse_signed_to_satoshi(s, denom).map_err(|error| error.convert(true))? {
             // (negative, amount)
-            (false, sat) if sat > SignedAmount::MAX.to_sat() as u64 => Err(ParseAmountError(
+            (false, sat) if sat > SignedAmount::MAX_MONEY.to_sat() as u64 => Err(ParseAmountError(
```

Comment:
My feeling is that `MAX` is more idiomatic, although I'm more opinionated that `MAX_MONEY` be left out of the docs.  Using it in the code doesn't matter as much, but also matching the docs isn't a bad thing.